owl-bot-staging/v1/protos/google/api/servicecontrol/v1/log_entry.proto

```diff
@@ -0,0 +1,126 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.api.servicecontrol.v1;
+
+import "google/api/servicecontrol/v1/http_request.proto";
+import "google/logging/type/log_severity.proto";
```

Comment:
I think this line is choking the Compiling protos... step.

The file `log_severity.proto` exists in neither this repo, this pull request, nor `googleapis/googleapis-gen`.

@alexander-fenster Owl Bot expects to find `log_severity.proto` under https://github.com/googleapis/googleapis-gen/tree/master/google/api/servicecontrol/v1/api-servicecontrol-v1-nodejs/protos but it's not there. Could you update the bazel build rule to also copy `log_severity.proto` into https://github.com/googleapis/googleapis-gen/tree/master/google/api/servicecontrol/v1/api-servicecontrol-v1-nodejs/protos?